dcos/http.py

```diff
@@ -106,8 +117,10 @@ def _request(method,
     :type url: str
     :param is_success: Defines successful status codes for the request
     :type is_success: Function from int to bool
-    :param timeout: request timeout
-    :type timeout: int
+    :param timeout: How many seconds to wait for the server to send data
+                    before giving up, as a float, a (connect timeout, read
+                    timeout) tuple or `True`. `None` means unlimited timeout.
+    :type timeout: True | None | float | (None | float, None | float)
```

Comment:
`True` is not a `type`. I believe that this should be `bool`, but I am not hugely familiar with this documentation style.

Reply:
I am also missing the exact documentation of the pydoc format we use, and work mostly by comparing the existing we have...

As we currently have `None` in some docs, and it is not a type either, I assume `True` is also valid.

Reply:
That is not a safe assumption. For example, `None` is shorthand for `type(None)` in `mypy`. `True`'s type is `bool` and that is the only way I have seen its type documented anywhere.